service/pager/query_pager.hh

```diff
@@ -21,10 +21,19 @@
 namespace service {
 
 class storage_proxy;
+class storage_proxy_coordinator_query_options;
 class storage_proxy_coordinator_query_result;
 
 namespace pager {
 
+using query_function = std::function<future<exceptions::coordinator_result<service::storage_proxy_coordinator_query_result>>(
+        service::storage_proxy& sp,
+        schema_ptr schema,
+        lw_shared_ptr<query::read_command> cmd,
+        dht::partition_range_vector&& partition_ranges,
+        db::consistency_level cl,
+        service::storage_proxy_coordinator_query_options optional_params)>;
+
```

Comment:
Another way to do it is to pass a different storage_proxy that talks to the local replica. We could make this part of the Great Storage Proxy Refactoring. /cc @xemul @kbr-scylla. Using this function is a good solution for now.